code/__DEFINES/traits.dm

```diff
@@ -325,6 +325,8 @@ Remember to update _globalvars/traits.dm if you're adding/removing/renaming trai
 #define TRAIT_CAN_USE_FLIGHT_POTION "can_use_flight_potion"
 /// This mob overrides certian SSlag_switch measures with this special trait
 #define TRAIT_BYPASS_MEASURES "bypass_lagswitch_measures"
+///  Someone can safely be attacked with honorbound, it won't hurt them.
+#define TRAIT_ALLOWED_HONORBOUND_ATTACK "allowed_honorbound_attack"
```

Comment:
Isn't the comment actually wrong? There's nothing in the code that actually prevents mobs with this trait from taking damage when attacked by a honorbounded character. It creates an exploit to circumvert `is_honorable()` by using a drape or bedsheet on a lying carbon mob.